add the feature to adjust exposure manually for MT9M114 camera on VIZ…

alan-n-n opened the pull request:
…N2_board

alan-n-n committed:
add the feature to adjust exposure manually for MT9M114 camera on VIZN2_board

Raymond0225 reviewed (commented):
revert changes on .cproject and lauch file

Raymond0225 commented on firmware_app/video-utilities/camera_rtvision.cpp:
```diff
@@ -441,7 +442,7 @@ static void Camera_RgbIrSwitch(int8_t cameraID)
         CAMERA_DEVICE_Start(&cameraDevice[cameraID]);
 
-#if (APP_CAMERA_TYPE == APP_CAMERA_GC0308)
+#if ((APP_CAMERA_TYPE == APP_CAMERA_GC0308) || (APP_CAMERA_TYPE == APP_CAMERA_MT9M114))
```
just remove this line, we have only these 2 camera type support.

Raymond0225 commented on firmware_app/video-utilities/camera_rtvision.cpp:
```diff
@@ -499,7 +500,7 @@ int Camera_SetDispMode(uint8_t displayMode)
 int Camera_SetExposureMode(uint8_t whichCamera, uint8_t mode)
 {
-#if (APP_CAMERA_TYPE == APP_CAMERA_GC0308)
+#if ((APP_CAMERA_TYPE == APP_CAMERA_GC0308) || (APP_CAMERA_TYPE == APP_CAMERA_MT9M114))
```
Remove this line

alan-n-n committed:
1. revert change to .cproject and .launch files
2. remove the macro definitions with GC0308 and MT9M114 cameras in
camera_rtvision.cpp as there are only 2 types

alan-n-n commented:
Hi, please review for my new commit
